DOCS-9263 sap successfactors 4.0 vh (#1108)
mulesoft/docs-connectors: sap-successfactors/4.0/modules/ROOT/pages/index.adoc

= SAP SuccessFactors Connector 4.0 - Mule 4
:page-aliases: connectors::sap/sap-successfactors-connector.adoc

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

Anypoint Connector for SAP SuccessFactors (SuccessFactors Connector) provides full support to query, create, update, and delete entities using the OData API exposed by SuccessFactors.

> This connector enables you to:
> 
> * Create, update, and delete entities
> * Retrieve a single entity by ID or query multiple entities

Reviewer: ```suggestion

```

Reviewer: Remove since it's already in the Common Use Cases for the Connector section

For compatibility information and fixed issues, see the SAP SuccessFactors Connector Release Notes.

== Prerequisites

To use this connector, you must be familiar with: 

* SAP SuccessFactors OData API
* Mule runtime engine (Mule)
* Anypoint Connectors
* Anypoint Studio essentials

> To use this connector you must have the following:
> 
> * Credentials to access a SuccessFactors developer account
> * A valid SAP SuccessFactors instance
> * A MuleSoft Enterprise license
> * Anypoint Studio 7.1 or later

Reviewer: ```suggestion
Before creating an app, you must have:

* Credentials to access a SuccessFactors developer account
* A valid SAP SuccessFactors instance
* A MuleSoft Enterprise license
* Access to Anypoint Studio 7.1 or later
```
